docs/patterns/flashing.rst

```diff
@@ -48,7 +48,7 @@ And here is the :file:`layout.html` template which does the magic:
    <title>My Application</title>
    {% with messages = get_flashed_messages() %}
      {% if messages %}
-       <ul class=flashes>
+       <ul class="flashes">
```

Comment:
quotes are optional

Reply:
true, and it's probably my pedanticism, but it's inconsistent across the example and I would argue double quotes are preferred, but fair enough